flow/compositor_context.h

```diff
@@ -20,7 +21,17 @@ namespace flutter {
 
 class LayerTree;
 
-enum class RasterStatus { kSuccess, kFailed };
+// kSuccess: frame has successfully rasterized.
+//
+// kResubmit: frame needs to be resubmitted for rasterization. This is currently
+// only called when thread configuration change occurs.
+//
+// kEnqueuePipeline: frame has been successfully rasterized, but pipeline
+// pressure needs to be re-applied. This is currently only called when thread
```

Comment:
nit: I wasn't sure what "pipleline pressure" meant, consider saying more explicitly "there are additional items in the pipeline waiting to be consumed"

Reply:
Done